lxd/device/device_utils_network.go

```diff
@@ -411,7 +412,8 @@ func networkNICRouteDelete(routeDev string, routes ...string) {
 			Proto:   "boot",
 			Family:  ipVersion,
 		}
-		r.Flush()
+
+		_ = r.Flush()
```

Comment:
This needs to be assigned to err otherwise the error check on the next line won't be checking if the removal failed (the `Flush()` function is what tries to do the route removal).